HW_1.sql

```diff
@@ -0,0 +1,69 @@
+-- Выберите все строки из таблицы suppliers. Предварительно подключитесь к базе данных northwind.
+USE northwind;
+SELECT 
+    *
+FROM
+    suppliers;
+
+-- Выберите только те строки из таблицы suppliers где company имеет значение Supplier A.
+SELECT 
+    *
+FROM
+    suppliers
+WHERE
+    company = 'Supplier A';
+
+-- Выберите все строки из таблицы purchase_orders.
+SELECT 
+    *
+FROM
+    purchase_orders;
+
+-- Выберите только те строки из таблицы purchase_orders,  где supplier_id = 2.
+SELECT 
+    *
+FROM
+    purchase_orders
+WHERE
+    supplier_id = 2;
+
+-- Выберите supplier_id и shipping_fee из purchase_orders там где created_by равно 1 и supplier_id равен 5.
+-- Объясните полученный результат.
+SELECT 
+    supplier_id, shipping_fee
```

Comment:
забыли добавить поле
```suggestion
    id, supplier_id, shipping_fee
```